tests/integration/models/reddit/test_subreddit.py

```diff
@@ -370,6 +370,84 @@ def test_submit_poll__live_chat(self, _):
             )
             assert submission.discussion_type == "CHAT"
 
+    def test_submit_gallery(self):
+        self.reddit.read_only = False
+        with self.recorder.use_cassette("TestSubreddit.test_submit_gallery"):
+            subreddit = self.reddit.subreddit(pytest.placeholders.test_subreddit)
+            images = [
+                {"image_path": self.image_path("test.png")},
+                {"image_path": self.image_path("test.jpg"), "caption": "test.jpg"},
+                {
+                    "image_path": self.image_path("test.gif"),
+                    "outbound_url": "https://example.com",
+                },
+                {
+                    "image_path": self.image_path("test.png"),
+                    "caption": "test.png",
+                    "outbound_url": "https://example.com",
+                },
+            ]
+
+            submission = subreddit.submit_gallery("Test Title", images)
+            assert submission.author == pytest.placeholders.username
+            assert submission.is_gallery
+            assert submission.title == "Test Title"
+            items = submission.gallery_data["items"]
+            assert isinstance(submission.gallery_data["items"], list)
+            for i, item in enumerate(items):
+                test_data = images[i]
+                test_data.pop("image_path")
+                item.pop("id")
+                item.pop("media_id")
+                assert item == test_data
+
+    def test_submit_gallery_disabled(self):
```

Comment:
What does `disabled` mean?

Reply:
If they try and submit to a sub that has galleries disabled.

Reply:
Oh cool. Is there anything useful in the exception that's raised that indicates that subreddit has galleries disabled?

Reply:
It returns this 
```json

{"json": {"errors": [["GALLERIES_DISABLED", "Gallery posts aren't allowed in this subreddit","sr"]]}}

```